vaccine_feed_ingest/runners/me/maine_gov/normalize.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import re
+import sys
+
+from vaccine_feed_ingest.schema import schema  # noqa: E402
+
+CITY_RE = re.compile(r"^([\w ]+), NY$")
+# the providerName field smells like it's being parsed from someplace else,
+# a good number of them have leading \u1d42 and/or *, which we want to clean.
+# there's a bunch with a city name in them, but no real pattern to it, so
+# we'll leave that for now.
+NAME_CLEAN_RE = re.compile("^[\u1d42*]+")
+
+logger = logging.getLogger(__name__)
+
+
+def _get_name(site: dict) -> str:
+    return site["providerName"]
+
+
+def _get_city(site: dict) -> str:
+    return site["city"]
+
+
+def _get_id(site: dict) -> str:
+    name = _get_name(site)
+    city = _get_city(site)
+    return f"me:maine_gov:{name}:{city}"
+
+
+def _get_source(site: dict, timestamp: str) -> schema.Source:
+    return schema.Source(
+        data=site,
+        fetched_at=timestamp,
+        fetched_from_uri="https://www.maine.gov/covid19/vaccines/vaccination-sites",
+        id=_get_id(site),
```

Comment:
Forgot one thing, can you have the ID (just in Source) match with whatever record ID the maine page gives? if it doesn't specify anything, feel free to conjoin the name and city like you did above without the `me:maine_gov` namespace :)

Reply:
Done.